doc/gov/chapters/chapter02.md

```diff
@@ -34,13 +34,8 @@ As an open-source community organization, CNTT has developed the following proce
   
   - **What happens once I get onboarded?** Once you have reviewed the CNTT onboarding information and have agreed to the perquisite terms and agreements associated with participating and contributing to the Task Force, and you have completed onboarding and now it will be your opportunity to join desired work streams. For an initial period after onboarding, you will be assigned a volunteer mentor from the Task Force to help you get up to speed and contributing quickly.
   
-  - **Overall, the process is as follows**
-  - 1. Request Membership
-  - 2. Review General Information packet
-  - 3. Agree to Terms
-  - 4. Join the Community
-  - 5. Begin Contributing
-  
+  - **Diagram**
+<p align="middle"><img src="../figures/onboarding_proc.png" alt="onboarding_proc" title="Onboarding Process" width="60%"/></p>
```

Comment:
The figure is not part of the pr. Please add it.

Reply:
Gergely, yea. Well, it's on a separate one. See #656

Reply:
> 
> 
> Gergely, yea. Well, it's on a separate one. See #656

Can you merge this two pr-s?

Reply:
@CsatariGergely @jbeltranatl I have moved the diagram into this PR. will resolve now.